zilencer/views.py

```diff
@@ -132,6 +136,31 @@ def register_remote_server(
     except ValidationError as e:
         raise JsonableError(e.message)
 
+    # We don't want to allow disposable domains for contact_email either
+    try:
+        validate_disposable(contact_email)
+    except DisposableEmailError:
+        raise JsonableError(_("Contact email in disposable domains is not allowed"))
```

Comment:
We can use `_("Please use your real email address.")` -- the same string used in account registration.